osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/ThdFingerprintConverterTest.java

```diff
@@ -75,8 +77,9 @@ void convertBundleWhenFingerprintNull() {
     assertThat(response.getThdFingerprint()).isNull();
   }
 
-  @Test
-  void convertSingle() {
+  @ParameterizedTest
+  @ValueSource(booleans = {true, false})
+  void convertSingle(final boolean polyphase) {
```

Comment:
`polyphase` is not used

Reply:
Did not finish my intention to test. Now I did.